app/models/monthly_statistics_report.rb

```diff
@@ -0,0 +1,24 @@
+class MonthlyStatisticsReport < ApplicationRecord
```

Comment:
I think I like the idea of having a row/record per table of the report because the different tables could be updated independently that way - if one table was flakey or slow to generate we could still generate all the others. Not a strong opinion though.

Reply:
@duncanjbrown discussed and think we agreed we liked the idea of having one row per report (kind of like a report snapshot). I don't think speed would be an issue but had not considered flakiness ... thoughts @duncanjbrown ?

Reply:
Good q. I think this isn't enough of a risk to warrant splitting it into multiple rows at this stage?

Reply:
No it's a minor thing. Lets stick with the all in one record model.